src/components/VmDetails/CardEditButton.js

```diff
@@ -40,30 +38,29 @@ class CardEditButton extends React.Component {
   }
 
   render () {
-    const { tooltip, id, editable, disableTooltip } = this.props
+    const { tooltip, id, editable, disableTooltip, placement = 'top' } = this.props
     const { editEnabled } = this.state
 
     const classes = `${style['card-edit-button']} ${style[editEnabled ? 'card-edit-button-enabled' : 'card-edit-button-disabled']}`
     const myClick = editEnabled ? noop : this.enableEditHandler
 
     if (!editable && disableTooltip) {
-      return <OverlayTrigger
-        overlay={<Tooltip><span>{disableTooltip}</span></Tooltip>}
-        placement='right'
-        trigger={['hover', 'focus']}
-        rootClose={false}
+      return (<OverlayTooltip
+        tooltip={<span>{disableTooltip}</span>}
+        placement={placement}
+        id={`${id}-card-edit-button-tooltip`}
       >
         <a className={`${style['card-edit-button']} ${style['card-edit-button-disabled']}`} id={id}>
           <Icon type='pf' name='edit' />
         </a>
-      </OverlayTrigger>
+      </OverlayTooltip>)
     }
     if (!editable) {
       return null
     }
 
     return (
-      <OverlayTooltip id={`${id}-tooltip`} tooltip={tooltip} placement='bottom'>
+      <OverlayTooltip id={`${id}-tooltip`} tooltip={tooltip} placement={placement}>
```

Comment:
We are not changing that, we are setting the value to be setted dynamically, instead using always 'bottom', we are using default value which is 'top' like `FieldLevelHelp` but if we wants it to be other value we just need to set the value explicitly.

Reply:
OK. Just a note that choosing the 'top' placement as the default is not clear to me. I don't see a reason to use it as the default because it's " like `FieldLevelHelp`". But I am okay with that because we probably want the placement in the top in most of the cases, in general :)